templates/channels/types/instagram/claim.haml

```diff
@@ -0,0 +1,136 @@
+-extends "channels/channel_claim_form.html"
+-load i18n compress temba
+
+-block pre-form
+  -blocktrans trimmed with name=brand.name
+    You can connect your Instagram business account to {{name}} in just a few simple steps.
+
+-block form
+  .mt-4.card
+    #fb-guide
+      %ol.steps
+        %li
+          -trans "The page must be linked to your instagram business account, so check how to link"
+          %a{:href => 'https://help.instagram.com/399237934150902'} here
```

Comment:
not familiar with this template syntax.. what is it and does it work?

Reply:
It's just including this link in the word "here" and it's working. As shown in the image below, in step 1.

![image](https://user-images.githubusercontent.com/30378788/149384607-899901ef-3c95-4bd4-9c3e-bdc31e354cb2.png)